### PR TITLE
Bfu warmup

### PR DESCRIPTION
**Pull request by B1s9l**
# Pull Requests Checklist

Assignee:

- \[ \] code commented?
- \[ \] deactivated things not needed? (e.g. other features which only were prepared)
  - \[ \] alerts that this is todo?
- \[ \] test run localhost?
- \[ \] tested manually all features and edge cases?
- \[ \] documentation added / updated?
- \[ \] pre-commit checks passed?
- \[ \] pipelines passed?

Reviewer:

- \[ \] correct branch?
- \[ \] are there any code smells?
- \[ \] did pipeline pass?
- \[ \] is the code well documented?
- \[ \] tested manually all features and edge cases?

### CODE REVIEW

File: src/components/views/Landing.tsx [hunk 1]
```diff
@@ -1,12 +1,31 @@
-import React from "react";
+import React, { useState } from "react";
```
Comment (5v3nn) [inline src/components/views/Landing.tsx:1]: use state not needed anymore

File: src/components/views/Landing.tsx [hunk 2]
```diff
@@ -1,12 +1,31 @@
...
+      } else {
+        console.log("Could not reach server. Try again!");
+      }
+    } catch (error) {
+      console.log("Could not reach server:", error);
+    }
+  };
 
   const handleLogoClick = () => {
+    setIsLoading(true);
```
Comment (5v3nn) [inline src/components/views/Landing.tsx:27]: we do not want to load

**Review by 5v3nn — CHANGES_REQUESTED** [review]

File: src/components/views/Landing.tsx (hunk 1 above)
Comment (B1s9l) [reply src/components/views/Landing.tsx]: Removed: done

File: src/components/views/Landing.tsx (hunk 2 above)
Comment (B1s9l) [reply src/components/views/Landing.tsx]: removed: done

**Review by 5v3nn — APPROVED** [review]
